### DISCUSSION TITLE
Debug: Command line arguments crash VS debugging session

### DISCUSSION THREAD

**Discussion by pmiossec** [gitextensions/gitextensions#10247] [Q&A]
In the file `launchSettings.json` has been committed the debug command line parameters `"browse \"$(MSBuildThisFileDirectory)..\""`.

I don't know if I'm the only one but most of the time (not always!) this debug setting prevent VS to launch the debugging session with the error "Object reference not set to an instance of an object"

![image](https://user-images.githubusercontent.com/460196/195170010-45f85b5e-7dc9-43ee-acdd-6f56ba517311.png)

A quick test seems to point toward the use of `$(MSBuildThisFileDirectory)`.
But I don't find anything on the web.

Does someone has the beginning of a clue? Could we change the value for  `"browse \"..\""` that behave the same?

VisualStudio 2022 is up to date (v17.3) and I already tried to reset or delete the settings without success.

**Comment by mstv** [comment]
Thank you! This works well for me.

**Comment by RussKie** [comment]
@pmiossec I had a similar issue and I worked around it by commenting out these lines:
https://github.com/gitextensions/gitextensions/blob/cb8954cc0cc839d7b33976e6befa492e5a6fde07/GitExtensions/GitExtensions.csproj#L24-L27

**Comment by gerhardol** [comment]
I delete `artifacts` folder every time after restarting MSVS.
Switching repos is fine.
